src/gui/mainwindow.cpp

```diff
@@ -805,6 +807,12 @@ void MainWindow::displayRSSTab() const
         m_tabs->setCurrentWidget(m_rssWidget);
 }
 
+void MainWindow::displayExecutionLogTab() const
+{
```

Comment:
Would expect that from a "display"  function. Maybe `activateExecutionLogTab()` or `focusExecutionLogTab()` are better names?